components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/description/query/SQLQuery.java

```diff
@@ -1391,7 +1418,7 @@ private PreparedStatement createProcessedPreparedStatement(int queryType,
             /* create a new prepared statement */
             if (stmt == null) {
                 /* batch mode is not supported for dynamic queries */
-                Object[] result = this.processDynamicQuery(this.getQuery(), params);
+                Object[] result = this.processDynamicQuery(query, params);
```

Comment:
Why use of getQuery is removed?

Reply:
**this.getQuery()** returns the query defined in the .dbs file, since the manipulated query should pass in **"processDynamicQuery"** method I've used query variable.